### DISCUSSION TITLE
how do I remove all the default allowed html and css tags ?

### DISCUSSION THREAD

**Discussion by raksha-shanbhag** [mganss/HtmlSanitizer#357] [Q&A] [answered]
^title

**Comment by mganss** [comment, marked as answer]
Do you mean disallow all elements that are allowed by default?

```c#
var s = new HtmlSanitizer();
s.AllowedTags.Clear();
s.AllowedTags.Add("b");
```

This will allow only the `<b>` element and disallow all others.

**Reply by raksha-shanbhag** [reply]
So this is going to clear everything ?

**Reply by mganss** [reply]
Yes

**Comment by navidyazdi** [comment]


**Reply by mganss** [reply]
It was the OP's original question 🤷🏻‍♂️

**Reply by navidyazdi** [reply]
Sorry for that, I confused between 2 questions in different tabs :\
